Setup/UpgradeData.php

```diff
@@ -163,5 +197,18 @@ public function upgrade(
             $lastSyncCodeId->setData('used_in_forms', ['adminhtml_customer',]);
             $lastSyncCodeId->getResource()->save($lastSyncCodeId);
         }
+
+        if (version_compare($context->getVersion(), '1.0.3', '<')) {
+            $configJson = $this->client->getResource('config');
```

Comment:
@ryanreeves-taxjar Do we need to check for an API token before hitting this endpoint?